frontend/javascripts/oxalis/model/sagas/proofread_saga.ts

```diff
@@ -378,9 +464,7 @@ function* splitOrMergeAgglomerate(action: MergeTreesAction | DeleteEdgeAction) {
 
   yield* call([api.data, api.data.reloadBuckets], layerName);
 
-  const volumeTracingWithEditableMapping = yield* select((state) =>
-    getActiveSegmentationTracing(state),
-  );
+  volumeTracingWithEditableMapping = yield* select((state) => getActiveSegmentationTracing(state));
```

Comment:
I think, you can remove this line. As I see it, you basically moved the initialization of this variable to the top of the saga, since you needed the mapping id and I think that's fine. However, it's not necessary to call `getActiveSegmentationTracing` here again in my opinion. Then, the variable can be a `const` again.